contracts/SnappAuctionChallenge.sol

```diff
@@ -126,13 +122,11 @@ contract SnappAuctionChallenge {
 
     function challengeStateTransition(
         bytes memory pricesAndVolumes,
-        bytes memory orders,
-        bytes memory stateRoots,
+        bytes memory orders,      // Should this be "challengeAuctionSettlement"?
+        bytes memory stateRoots,  // What are these and how many are they?
```

Comment:
Intermediate state roots (also called checkpoints) of the transition. Right now I envision ~5 orders between checkpoints --> ~200 intermediate state roots for 1k orders/solution.